### ISSUE TITLE
tag added tracks with username

### ISSUE THREAD

**Issue by hamilton5** [azlux/botamusique#333]
**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**
tag track as the user that added it; ie u: hamilton5

**Describe alternatives you've considered**
manual editing

**Additional context**
the logs are hard to follow, especially with duplicate entries, and the built in limits don't help either. logs are small an gone fast if you play shuffle constantly. who added it and the ability to add all of 'your' tracks to the playlist would be nice.

**Comment by TerryGeng** [comment]
Yes, this sounds like a good use of the tag system. But mind you, the user who adds the song can also edit that tag. Do you think hiding that tag would be a good idea?

**Comment by hamilton5** [comment]
Not sure. If it was hidden then no one could see it/use it or know who added it? I'm more interested in the logging purposes, but then thought it would be cool to be able to add all songs from a user.
